packages/main/src/DayPicker.hbs

```diff
@@ -48,12 +40,20 @@
 											{{this.iDay}}
 									</span>
 							</div>
-						{{/each}}
-					</div>
-				{{else}}
-					<div class="sapWCEmptyWeek"></div>
-				{{/if}}
-			{{/each}}
-		</div>
+						{{else}}
+							{{#unless _hideWeekNumbers}}
```

Comment:
After this change the hideWeeknumbers property does no longer work.

Reply:
done